### ISSUE TITLE
Generate Operator manifests with jsonnet

### ISSUE THREAD

**Issue by squat** [observatorium/deployments#301]
Currently, all of the manifests for deploying the operator, ie the RBAC and deployment, are written by hand. These should be generated via jsonnet just like all other observatorium components.

**Comment by brancz** [comment]
Agreed! A slight extension in the permissions part (could definitely be done as a follow up to this), I always imagined, that locutus would be able to generate the RBAC roles necessary based on it's input and the configured/potential actions performed and informers configured.
